### PR TITLE
Update Click Event

### PR DESCRIPTION
**Pull request by michaelwehrley**
Update Click Event

* Update page json payload.
* Add attributes to `ClickEvent` to display meta data and if page clicked was accessible.

[Finishes: #94627446]

### CODE REVIEW

File: app/models/click_event.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ class ClickEvent < ActiveRecord::Base
             presence: true
 
   # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
   def self.deserialize(event_clicks)
     event_clicks.each_with_object([]) do |click, array|
```
Comment (ericcf) [inline app/models/click_event.rb]: Any reason you're not using `.map` here?

Comment (michaelwehrley) [inline app/models/click_event.rb]: yeah, because of `clicked_at: (DateTime.parse(click["clickedAt"]) if click["clickedAt"]),`

Comment (ericcf) [inline app/models/click_event.rb]: Hmm...I don't understand. Can you elaborate?

Comment (michaelwehrley) [inline app/models/click_event.rb]: Ahh, you mean instead of `each_with_object`? I will try.

**Comment by michaelwehrley** [comment]
@ericcf updates were made.  Thanks.

**Comment by ericcf** [comment]
:+1:

**Pull request merged**